src/atomate2_temp/aims/files.py

```diff
@@ -62,12 +58,10 @@ def copy_aims_outputs(
     files = ["aims.out", "*.json"]
 
     for pattern in set(additional_files):
-        for f in (glob((Path(src_dir) / pattern).as_posix())):
```

Comment:
I usually write this as `for f in (Path(src_dir) / pattern).glob('*'):` to avoid unnecessary type conversion

Reply:
I don't think I modified the file outside of maybe black stuff